lib/validation/schema/specVersion/kind/extension.json

```
@@ -70,6 +70,22 @@
 					"type": "string"
 				}
 			}
+		},
+		"metadata-3.0": {
+			"type": "object",
+			"required": ["name"],
+			"additionalProperties": false,
+			"properties": {
+				"name": {
+					"type": "string",
+					"minLength": 3,
+					"maxLength": 50,
+					"pattern": "^(?:@[0-9a-z-_.]+/)?[a-z][0-9a-z-_.]*$"
```

```suggestion
					"pattern": "^(?:@[0-9a-z-_.]+\/)?[a-z][0-9a-z-_.]*$"
```

Escape the backslash

---

Done